hnn_core/cell_response.py

```diff
@@ -423,6 +425,33 @@ def write(self, fname):
                         int(self._spike_gids[trial_idx][spike_idx]),
                         self._spike_types[trial_idx][spike_idx]))
 
+    def write_hdf5(self, fname):
```

Comment:
same comment, triage based on file extension. However, raise a deprecation warning if the file extension is txt.